src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Color.txt

```diff
@@ -0,0 +1,56 @@
+﻿>> ColorValue("#01020304")
+RGBA(1,2,3,0.016)
+
+>> ColorValue("#010203")
+RGBA(1,2,3,1)
+
+>> ColorValue("#aabbccdd")
+RGBA(170,187,204,0.867)
+
+>> ColorValue("#aabbcc")
+RGBA(170,187,204,1)
+
+>> ColorValue("#AABBCCDD")
+RGBA(170,187,204,0.867)
+
+>> ColorValue("#AABBCC")
+RGBA(170,187,204,1)
+
+>> ColorValue("#ffffffff")
+RGBA(255,255,255,1)
+
+>> ColorValue("#FFFFFFFF")
+RGBA(255,255,255,1)
+
+>> ColorValue("#00000000")
+RGBA(0,0,0,0)
+
+>> ColorValue("#000000")
+RGBA(0,0,0,1)
+
+>> ColorValue("Red")
```

Comment:
>Red [](http://example.com/codeflow?start=15&length=3)

Case insensitive? 
What aobut the other color vlaues? 

What about unsupported color names (which don't star with # )

Reply:
Correct, case insensitive, for other colors it would give an error. Added test for the same.